### PR TITLE
Build: Remove examples from workspace (old)

### PR DESCRIPTION
**Pull request by shilman**
Issue: N/A

## What I did

- [x] Remove examples from workspace to speed up install time
- [x] Fix resulting build errors
- [ ] Fix CI
- [ ] Fix package publishing

## How to test

- [ ] Does CI pass?

### CODE REVIEW

**Comment by gaetanmaisse** [comment]
@merceyz what we also have in mind with this move is to use PnP for some of the examples, at least CRA and Web Components. I would think using PnP + portal protocol should help us:
 - to be closer to end users' setups (with less hoisting related to the monorepo)
 - to ensure Storybook is working in PnP mode because we should be able to test/use it easily and not only in our e2e

WDYT? Is it an interesting _first_ step?

**Comment by merceyz** [comment]
Sure, seems reasonable, go for it 👍

File: examples/cra-kitchen-sink/.yarnrc.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+enableGlobalCache: true
```
Comment (merceyz) [inline examples/cra-kitchen-sink/.yarnrc.yml:1]: Settings are inherited from all parent folders so this file is redundant

**Comment by ndelangen** [comment]
@shilman please re-open if you think it's useful, but I've heard to talk about another PR replacing this. I think we got the knowledge we needed out of this PR, but it's extremely unlikely to ever be merged. Again: please re-open if I'm wrong here!

**Pull request closed without merging**